infoportal-server/src/feature/kobo/meta/KoboMetaMapperShelter.ts

```diff
@@ -96,6 +96,27 @@ export namespace KoboMetaMapperShelter {
     const answer = Shelter_cashForShelter.map(row.answers)
     const group = KoboGeneralMapping.collectXlsKoboIndividuals(answer).map(KoboGeneralMapping.mapPerson)
     const oblast = OblastIndex.byKoboName(answer.ben_det_oblast!)
+    const projects: DrcProject[] = [];
+    if (answer.donor) {
+      const donorIds: string[] = answer.donor.split(',');
```

Comment:
Should not be needed, `Shelter_cashForShelter.map` already takes care of converting as array options if relevant. As shown in the file above here `donor: undefined | Option<'donor'>,`, this is a select simple.